Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG USE_VENDOR=false
```

Comment:
Enabled/Disabled should be better than true/false

Reply:
Ok, I will update this.